### PR TITLE
Add NTO Operator and avoid crashing on invalid test specs

### PR DESCRIPTION
**Pull request by kpouget**
image

### CODE REVIEW

**Comment by kpouget** [comment]
@jmencak here it comes :)

File: Makefile [hunk 1]
```diff
@@ -28,5 +35,22 @@ output/sro_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
...
              --output-file $@
 
+# NTO
+
+nto: output/nto_daily-matrix.html output/nto_daily-matrix.md
+
+output/nto_daily-matrix.html: templates/daily_matrix.tmpl.html
+	go run cmd/main.go --debug daily_matrix \
+	     --config-file examples/nto.yml \
+             --template  $< \
```
Comment (jmencak) [inline Makefile]: `s/--template  $</--template $</`

File: Makefile [hunk 2]
```diff
@@ -28,5 +35,22 @@ output/sro_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
...
+             --output-file $@
+
+output/nto_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
+	go run cmd/main.go --debug daily_matrix \
+	     --config-file examples/nto.yml \
```
Comment (jmencak) [inline Makefile]: Formatting, the same indentation with all the options, please.

File: Makefile [hunk 3]
```diff
@@ -28,5 +35,22 @@ output/sro_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
...
+             --template  $< \
```
Comment (jmencak) [inline Makefile]: `s/--template $</--template $</`

**Review by jmencak — COMMENTED** [review]
This is great, thanks for doing this, Kevin. Until I figure out how to test this, only nits...

File: Makefile (hunk 1 above)
Comment (kpouget) [reply Makefile]: not sure what you mean here, I see twice the same thing ;-) (and Firefox's `ctrl-f` as well)

File: Makefile (hunk 3 above)
Comment (kpouget) [reply Makefile]: there must be some wind in here, every line if further right than the previous one ... I'm not sure how I managed to mess indentation like that!

File: Makefile (hunk 1 above)
Comment (kpouget) [reply Makefile]: hum, got it, double spaces

**Comment by kpouget** [comment]
@jmencak nits should be fixed

> Until I figure out how to test this, only nits...

part of the testing is automated as a Github Action, it ensures that the pages can be generated without error

what I can't do yet is store and display the generated pages for visual inspection, I don't know if there's a way to trick Github Actions to do it, it would be good for `ci-artifacts` documentation as well ...

**Comment by jmencak** [comment]
The required test is failing with `Incorrect Usage: flag provided but not defined: -debug` sounds like an option parsing issue.

**Comment by kpouget** [comment]
> The required test is failing with `Incorrect Usage: flag provided but not defined: -debug` sounds like an option parsing issue.

I tried to move `--debug` along with the options, but the syntax is `go run cmd/main.go [--debug] daily_matrix [<daily_matrix options>]`, and `--debug` doesn't belong to `daily_matrix options`!

**Comment by kpouget** [comment]
https://openshift-psap.github.io/ci-dashboard/nto_daily-matrix.html \o/

**Pull request merged**